src/Styleguide/Components/RecentlyViewed.tsx

```diff
@@ -105,13 +106,7 @@ export const RecentlyViewedQueryRenderer = () => {
                 }
               }
             `}
-            render={({ props }) => {
-              if (props) {
-                return <RecentlyViewedFragmentContainer me={props.me} />
-              } else {
-                return null
-              }
-            }}
+            render={renderWithLoadProgress(RecentlyViewedFragmentContainer)}
```

Comment:
Noticed a few spots lately that could use this

Reply:
Ah right, I had intended to cc you on me adding this to `RecentlyViewed`. Yeah we should use it in more places 👍